core/logger/logger.go

```diff
@@ -191,60 +193,13 @@ func (l *zapLogger) NewServiceLevelLogger(serviceName string, logLevel string) (
 		return nil, err
 	}
 
-	config := initLogConfig(l.dir, l.jsonConsole, ll, l.toDisk)
-
-	zl, err := config.Build(zap.AddCallerSkip(1))
+	zl, err := initLogConfig(l.dir, l.jsonConsole, ll, l.toDisk).Build()
```

Comment:
These caller skips are no longer necessary with the default funcs and conditionals skipping properly, and they were broken on direct calls.